Index: pages/purchase/purchaseSubscribe.vue
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- pages/purchase/purchaseSubscribe.vue	(date 1574843438000)
+++ pages/purchase/purchaseSubscribe.vue	(date 1574843438000)
@@ -1,345 +1,21 @@
 <template>
   <article id="purchaseSubscribe">
     <header class="section-visual">
-      <h1>9900대치동 과외</h1>
+      <h1 class="hidden">9900대치동 과외</h1>
       <img src="/elementary/resources/img/membership/pc/visual_tit.png" alt="9900원 대치동 초등수학 과외">
-      <div class="btn-box">
-        <a class="btn" href="javascript:;" onclick="buy_membership('m');">
-          <img alt="지금 7일 무료체험하기" src="/elementary/resources/img/membership/pc/visual_btn01.png">
-        </a>
-        <a class="btn" href="javascript:;" onclick="buy_membership('3m');">
-          <img alt="3개월 정기결제 하기" src="/elementary/resources/img/membership/pc/visual_btn02.png">
-        </a>
-      </div>
+      <a class="btn" href="javascript:;" onclick="buy_membership('m');">
+        <img alt="지금 7일 무료체험하기" src="/elementary/resources/img/membership/pc/visual_btn01.png">
+      </a>
+      <a class="btn" href="javascript:;" onclick="buy_membership('3m');">
+        <img alt="3개월 정기결제 하기" src="/elementary/resources/img/membership/pc/visual_btn02.png">
+      </a>
     </header>
-<!--    <section class="section-feature">
-      <div class="in">
-        <h1>
-          <img src="/elementary/resources/img/membership/pc/feature_tit.png" alt="[초등학습 최초]어디에도 없던 초대박 구성 신청하면 무조건 이득!">
-        </h1>
-      </div>
+    <section class="section-feature">
+      <h1 class="hidden">
+        [초등학습 최초]어디에도 없던 초대박 구성 신청하면 무조건 이득!
+      </h1>
+      <img src="/elementary/resources/img/membership/pc/feature_tit.png" alt="[초등학습 최초]어디에도 없던 초대박 구성 신청하면 무조건 이득!">
     </section>
-    <div class="section-guide">
-      <div class="l-box">
-        <a href="/elementary/ip/9900fnq">
-          <img src="/elementary/resources/img/membership/pc/guide_7day.jpg" alt="1개월 정기결제 수강 가이드">
-        </a>
-      </div>
-      <div class="r-box">
-        <a href="/elementary/ip/9900fnq?months=3" target="_blank">
-          <img src="/elementary/resources/img/membership/pc/guide_3months.jpg" alt="3개월 정기결제 수강 가이드">
-        </a>
-      </div>
-    </div>
-    <div class="section-benefit01">
-      <div class="in">
-        <img src="/elementary/resources/img/membership/pc/benefit_01.png" alt="01주 1회 1시간 대치동 수학수업(월 4회)을 9900원에!">
-        <a id="pop_video" class="btn" href="#" ovideo_data="M8F_xnReWD8">
-          <img src="/elementary/resources/img/membership/pc/benefit01_btn.png" alt="3개월 정기결제 수강 가이드">
-        </a>
-      </div>
-    </div>
-    <div class="section-benefit02">
-      <div class="in">
-        <img src="/elementary/resources/img/membership/pc/benefit_02.png" alt="'02대치동 초등수학 20년 노하우의 콘텐츠 무료제공">
-      </div>
-    </div>
-    <div class="section-benefit02-sub">
-      <div class="in">
-        <img src="/elementary/resources/img/membership/pc/benefit_02_sub.png" alt="수학만 있다? NO! 전과목 교재 구매도 가능합니다!">
-        <a href="http://book.conects.com/event/eventDetail?evNo=14480" class="btn">
-          <img src="/elementary/resources/img/membership/pc/benefit02_sub_btn.png" alt="교재 자세히 보기">
-        </a>
-      </div>
-    </div>
-    <div class="section-benefit03">
-      <div class="in">
-        <img src="/elementary/resources/img/membership/pc/benefit_03.png" alt="03최고의 온라인 강좌들이 모두 무제한!">
-      </div>
-    </div>
-    <div class="section-review">
-      <div class="in">
-        <img src="/elementary/resources/img/membership/pc/review_tit.png" alt="생생 후기">
-        <div class="slide-box">
-          <div class="bx-wrapper" style="max-width: 1055px;">
-            <div class="bx-viewport" style="width: 100%; overflow: hidden; position: relative; height: 200px;">
-              <ul style="width: 8215%; position: relative; transition-duration: 20s; transform: translate3d(-2560px, 0px, 0px); transition-timing-function: linear;">
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;">
-                  <strong>
-                    선생님과 피드백이 바로바로 되니까 아이가 더 집중할 수 있어요~ 일간대치동 대만족입니다!
-                  </strong>
-                  <span>다래** 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;">
-                  <strong>
-                    0 하나 빠진줄 알았어요. 대치동 과외를 9900원에 받는다니 너무 파격적인거같아요!
-                  </strong>
-                  <span>빨간** 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;">
-                  <strong>
-                    진짜 좋은 기회네요. 9900원으로 수학과외도 듣고 다른 강좌도 들을 수 있다니 대박입니다.
-                  </strong>
-                  <span>롱* 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;">
-                  <strong>
-                    9900원 가격에 한 번 놀라고 아이가 너무 재미있어하고 수업시간을 기다리는 모습에 놀랐어요!
-                  </strong>
-                  <span>영주* 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;">
-                  <strong>
-                    친구들과 함께 수업하니 경쟁심도 생기고 수업에 더 열심히 참여한답니다~ 교재도 참신한 문제들이 많아요!
-                  </strong>
-                  <span>다래** 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;">
-                  <strong>
-                    감당힘든 사교육비의 현실, 9900원 응원합니다. 지속되길 바랍니다.
-                  </strong>
-                  <span>누구** 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;">
-                  <strong>
-                    대치동 노하우의 소수정예 방식이라니! 게다가 온가족이 함꼐 할 수 있는 혜택까지~ 좋습니다.
-                  </strong>
-                  <span>박** 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;">
-                  <strong>
-                    강의의 퀄리티와 가격까지 두 마리 토끼를 한 번에 잡은 기분이네요. 정말 최고입니다.
-                  </strong>
-                  <span>정**님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;" class="bx-clone">
-                  <strong>
-                    선생님과 피드백이 바로바로 되니까 아이가 더 집중할 수 있어요~ 일간대치동 대만족입니다!
-                  </strong>
-                  <span>다래** 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;" class="bx-clone">
-                  <strong>
-                    0 하나 빠진줄 알았어요. 대치동 과외를 9900원에 받는다니 너무 파격적인거같아요!
-                  </strong>
-                  <span>빨간** 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;" class="bx-clone">
-                  <strong>
-                    진짜 좋은 기회네요. 9900원으로 수학과외도 듣고 다른 강좌도 들을 수 있다니 대박입니다.
-                  </strong>
-                  <span>롱* 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;" class="bx-clone">
-                  <strong>
-                    9900원 가격에 한 번 놀라고 아이가 너무 재미있어하고 수업시간을 기다리는 모습에 놀랐어요!
-                  </strong>
-                  <span>영주* 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;" class="bx-clone">
-                  <strong>
-                    친구들과 함께 수업하니 경쟁심도 생기고 수업에 더 열심히 참여한답니다~ 교재도 참신한 문제들이 많아요!
-                  </strong>
-                  <span>다래** 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;" class="bx-clone">
-                  <strong>
-                    감당힘든 사교육비의 현실, 9900원 응원합니다. 지속되길 바랍니다.
-                  </strong>
-                  <span>누구** 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;" class="bx-clone">
-                  <strong>
-                    대치동 노하우의 소수정예 방식이라니! 게다가 온가족이 함꼐 할 수 있는 혜택까지~ 좋습니다.
-                  </strong>
-                  <span>박** 님</span>
-                </li>
-                <li style="float: left; list-style: none; position: relative; width: 310px; margin-right: 10px;" class="bx-clone">
-                  <strong>
-                    강의의 퀄리티와 가격까지 두 마리 토끼를 한 번에 잡은 기분이네요. 정말 최고입니다.
-                  </strong>
-                  <span>정**님</span>
-                </li>
-              </ul>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-    <div class="section-event" id="section-event">
-      <div class="in">
-        <img src="/elementary/resources/img/membership/pc/event_tit.png" alt="">
-        <div class="nested-vue-app">
-          <div data-v-010a030e="" class="the-comments">
-            <div data-v-010a030e="" class="pass-comment-board-desktop" id="the-comments">
-              <div class="board-list">
-                <div class="board-list desktop">
-                  <table>
-                    <colgroup>
-                      <col style="width: 10%;">
-                      <col style="width: 80%;">
-                      <col style="width: 10%;">
-                    </colgroup>
-                    <colgroup>
-                      <col style="width: 20%;">
-                      <col style="width: 60%;">
-                      <col style="width: 20%;">
-                    </colgroup>
-                    <tbody>
-                    <tr>
-                      <td class="desktop board-content nick-name ">일간대치동</td>
-                      <td class="board-content content desktop">@마녀님 대치동 아이들만 받던 그 수업, 전국 모든 아이들도 다 함께 들을 수 있도록 정말 많이 준비했답니다~ 믿고 맡겨주세요 : )</td>
-                      <td class="board-content date desktop">2019 . 11 . 25</td>
-                    </tr>
-                    <tr>
-                      <td class="desktop board-content nick-name ">일간대치동</td>
-                      <td class="board-content content desktop">@갓갓님 10명 중 9명의 아이들이 먼저 신청해달라고 하는 바로 그 수업! 지금 바로 들어보세요 :D</td>
-                      <td class="board-content date desktop">2019 . 11 . 25</td>
-                    </tr>
-                    <tr>
-                      <td class="desktop board-content nick-name ">일간대치동</td>
-                      <td class="board-content content desktop">@설레는향님 1:4 그룹수업으로 이루어지는 발표 및 토록 최적화 수업! 게다가 매 수업이 끝나자마자 우리 아이 공부 현황까지 알 수 있는 학습리포트까지~ 이 모든 것을 9900원에 만나보세요 : )</td>
-                      <td class="board-content date desktop">2019 . 11 . 25</td>
-                    </tr>
-                    <tr>
-                      <td class="desktop board-content nick-name ">마녀</td>
-                      <td class="board-content content desktop">아이가 너무너무 좋아하네요 큰 도움이 되었어요!!! 감사합니다 ^^^%</td>
-                      <td class="board-content date desktop">2019 . 11 . 23</td>
-                    </tr>
-                    <tr>
-                      <td class="desktop board-content nick-name ">갓갓</td>
-                      <td class="board-content content desktop">이렇게라도 아이들 공부를 시킬 수 있어서 다행이에요 ㅠ 정말 최고입니다!</td>
-                      <td class="board-content date desktop">2019 . 11 . 23</td>
-                    </tr>
-                    </tbody>
-                  </table>
-                </div>
-              </div>
-              <div class="board-pagination board-pagination">
-                <ul>
-                  <li>
-                    <button disabled="disabled" class="disabled"><i class="glyphicon glyphicon-menu-left"></i></button>
-                  </li>
-                  <li class="select">1</li>
-                  <li class="">2</li>
-                  <li class="">3</li>
-                  <li class="">4</li>
-                  <li>
-                    <button class=""><i class="glyphicon glyphicon-menu-right"></i></button>
-                  </li>
-                </ul>
-              </div>
-              <div class="board-from">
-                <div class="private-info-input-area"><input id="nickName" type="text" placeholder="닉네임" maxlength="7"><label for="phone1">휴대폰 번호</label><input type="text" id="phone1" maxlength="3" class="phone-number-input"> -
-                  <input type="text" id="phone2" maxlength="4" class="phone-number-input"> -
-                  <input type="text" id="phone3" maxlength="4" class="phone-number-input"><span>*입력하신 번호로 경품이 발송될 예정이니, 정확하게 입력해주시기 바랍니다.</span></div>
-                <div class="private-info-check-area"><p class="private-info-title">개인정보 제공 동의 (필수)</p>
-                  <div class="private-info-content">
-                    <ul>
-                      <li>- 수집하는 개인정보의 항목
-                        <br>연락처
-                      </li>
-                      <li>- 개인정보를 제공받는 자 : [(주)에스티유니타스]</li>
-                      <li>- 개인정보를 제공받는 자의 개인정보 이용 목적 : 본인 확인 및 상품발송 알림</li>
-                      <li>- 개인정보를 제공받는 자의 개인정보 보유 및 이용기간 : 이용목적 달성 시까지</li>
-                      <li>- 개인 정보 수집에 대해 동의하지 않으실 경우, 서비스 이용에 제한이 있을 수 있습니다.</li>
-                    </ul>
-                  </div>
-                  <div class="private-info-check">
-                    개인정보 수집에 동의하시겠습니까?
-                    <input id="privateInfoCheck" type="checkbox"><label for="privateInfoCheck"><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAYAAABWzo5XAAAAlklEQVQ4je3UMQ4BYRRF4Q8TjWpGorMEnV1QqSm1VHQ2wBYUElsgNkFlF5MYUYhqEsUMneKPTpz+nLzX3Ep7c6tjhSESYVyxxSzCEpPAwIu4dB8RRrDvN3SataDKMc0NDncYV5XvhEag23o7STXY/sA/9A/9biiD8yUPlk/p28kixZ5Me7v7NwetI8xRUcxJHBh4DdviCRBJHKe5L9RBAAAAAElFTkSuQmCC" alt="동의버튼">
-                    동의합니다
-                  </label></div>
-                </div>
-                <div class="board-input-area"><input type="text" placeholder="9,900원 대치동 과외에 대한 자유로운 의견을 남겨주세요!">
-                  <button>저장하기</button>
-                  <div class="board-length-area">
-                    30자 이상 입력해주세요 (<span class="red-number">0</span>/30) *(욕설 및 비방 댓글은 무통보 삭제됩니다)
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-    <div class="section-plus">
-      <div class="in">
-        <h2>
-          <img src="/elementary/resources/img/membership/pc/plus_tit.png" alt="PLUS 놓치지 말아야 할 일간대치동 소식">
-        </h2>
-        <div class="btn-box">
-          <a href="/elementary/ip/junior_event_november">
-            <img src="/elementary/resources/img/membership/pc/plus_btn01.png" alt="업계최초 150% 환급 이벤트 보러가기">
-          </a>
-          <a href="/edu/ip/1on1class">
-            <img src="/elementary/resources/img/membership/pc/plus_btn02.png" alt="1:1과외 살펴보러 가기">
-          </a>
-        </div>
-      </div>
-    </div>
-    <div class="section-visual-bottom">
-      <div class="in">
-        <h2>
-          <img src="/elementary/resources/img/membership/pc/visual_bottom_tit.png" alt="9900원 대치동 초등수학 과외">
-        </h2>
-        <div class="btn-box">
-          <a class="btn" href="javascript:;" onclick="buy_membership('m');">
-            <img alt="지금 7일 무료체험하기" src="/elementary/resources/img/membership/pc/visual_btn01.png">
-          </a>
-          <a class="btn" href="javascript:;" onclick="buy_membership('3m');">
-            <img alt="3개월 정기결제 하기" src="/elementary/resources/img/membership/pc/visual_btn02.png">
-          </a>
-        </div>
-      </div>
-    </div>
-    <div class="section-notice">
-      <h3>유의사항</h3>
-      <div class="in">
-        1. 9900원 대치동 초등 과외_ 1개월 정기결제 상품<br>
-        <br>
-        * 결제 및 해지<br>
-        - 7일 무료체험 기간에 해지하시면 별도 과금이 이루어지지 않습니다.<br>
-        - 결제는 7일 무료 체험이 종료되는 날 자동 진행됩니다. (무료체험 신청 후 7일 경과)<br>
-        - 첫 결제 발생 이후, 30일 단위로 등록하신 카드로 자동 결제가 진행됩니다.<br>
-        - 정기결제 해지는 고객센터 1670-1485로 전화주시면 해지가 가능합니다.<br>
-        <br>
-        2. 9900원 대치동 초등 과외_ 3개월 정기결제 상품<br>
-        <br>
-        * 결제 및 해지<br>
-        - 3개월 정기결제 상품은 10% 할인 혜택이 제공되는 대신, 7일 무료체험 기간이 존재하지 않습니다.<br>
-        - 최초 결제 이후 90일 단위로 등록하신 카드로 자동 결제가 진행됩니다.<br>
-        - 정기결제 해지는 고객센터 1670-1485로 전화주시면 해지가 가능합니다.<br>
-        <br>
-        3. 환불 규정<br>
-        <br>
-        A) 정기 결제 시작 후, 7일 이내에 콘텐츠를 이용하지 않는 경우에만 전액 환불이 가능합니다.<br>
-        정기 결제 시작 후, 7일이 경과되거나 또는 콘텐츠를 이용한 경우에는 환불이 불가능하며, 다음 달 자동 결제부터 해지됩니다.<br>
-        자동결제 후 7일 이내에 콘텐츠를 이용한 경우에는 자동결제 해지 예약을 할 수 있습니다.<br>
-        * 콘텐츠를 이용한 경우는 다음과 같습니다.<br>
-        - 강의 : 해당 기간 동안 1강이라도 수강한 경우<br>
-        - 모임 : 해당 기간 동안 1개의 모임이라도 가입, 참여한 경우<br>
-        - 자료 : 해당 기간 동안 1개의 자료라도 보관함에 담은 경우<br>
-        - 수업권 : 지급된 수업권을 사용하여 수강신청을 한 경우
-      </div>
-    </div>
-    <div class="popup-video-box">
-      <div class="black_dim"></div>
-      <div class="pop-video-wrap">
-        <div class="view-box"></div>
-        <a class="pop-btn-close" href="#" id="pop_video_close">닫기</a>
-      </div>
-    </div>-->
-    <!-- 하단 띠배너 -->
-
-    <!--<div class="section_ban_bottom">
-      <div class="in">
-        <img src="/elementary/resources/img/membership/pc/ban_bottom-1.png" alt="하단 띠배너">
-        <a class="btn1" href="javascript:;" onclick="buy_membership('m');"></a>
-        <a class="btn2" href="javascript:;" onclick="buy_membership('3m');"></a>
-      </div>
-    </div>-->
-    <!-- //하단 띠배너 -->
   </article>
 </template>
 
@@ -404,13 +80,45 @@
   }
 }
 </script>
+<style lang="scss">
+/*레이아웃 영역*/
+#wrap {
+  width: 95rem;
+  margin: 0 auto 0 0;
+}
 
+.btn {
+  display: inline-block;
+  margin-bottom: 0;
+  font-weight: normal;
+  text-align: center;
+  vertical-align: middle;
+  touch-action: manipulation;
+  cursor: pointer;
+  background-image: none;
+  border: 1px solid transparent;
+  white-space: nowrap;
+  padding: 0.6rem 1.2rem;
+  font-size: 1.4rem;
+  line-height: 1.428571429;
+  border-radius: 0.4rem;
+  user-select: none;
+}
+
+.hidden {
+  position: absolute;
+  clip: rect(1px, 1px, 1px, 1px);
+}
+</style>
 <style lang=scss>
 .content-box {
   width: 100% !important;
 }
 
-#daechi_9900 {
+#purchaseSubscribe {
+  img {
+    width: 100%;
+  }
 }
 
 #daechi_9900 #day-timer {
